source/_docs/automation/trigger.markdown

```diff
@@ -422,7 +422,7 @@ The `for` template(s) will be evaluated when an entity changes as specified.
 
 <div class='note warning'>
 
-Use quotes around your values for `from` and `to` to avoid the YAML parser from interpreting values as booleans.
+Use quotes around boolean values (`Y`, `true`, `Yes`, `ON` and `N`, `FALSE`, `No`, `off`) for `from` and `to` to avoid the YAML parser from interpreting values as booleans. See [Booleans](https://www.home-assistant.io/docs/configuration/yaml/#booleans).
```

Comment:
The Home Assistant state engine, only works with strings for states. Thus using native boolean values doesn't make sense / is actually wrong.

Reply:
yes, and thats why they need to be quoted? 
I am not sure I follow what you are saying Frenck, do you want me to use a different terminology? Or do you want me to add that sentence, which I think would be a perfect (and educational) addition.

(I was merely triggered by the fact this warning is displayed under an example using quotes on a non boolean string, thus being confusing itself..)

Reply:
> yes, and thats why they need to be quoted?

Yes

> I am not sure I follow what you are saying Frenck, do you want me to use a different terminology?

I'm raising the issue that this change is incorrect IMHO, should not be merged; and giving you the opportunity to discuss it.

Reply:
right. thanks, right now it says quotes are (always) needed, to prevent the parser to interpret values as booleans. However, this is _only_ necessary when using 'boolean' values. Any other, even home/not_home, do not require quotes.

Since I only took the additional text from another HA documentation page (and I linked that), I am not sure what you see as incorrect at all.